ppb/mouse.py

```diff
@@ -0,0 +1,53 @@
+from typing import Callable
+
+from ppb.vector import Vector
+
+
+class Mouse:
+    """
+    Object interface to the hardware mouse.
+
+    """
+    def __init__(self, press_function: Callable=None,
+                 move_function: Callable=None, move_to_function: Callable=None):
+        self.position = Vector(0, 0)
+        self.screen_position = Vector(0, 0)
+        self.buttons = [False, False, False]
+
+        self.press_function = press_function or _not_implemented
+        self.move_function = move_function or _not_implemented
+        self.move_to_function = move_to_function or _not_implemented
+
+    @property
+    def left_button(self) -> bool:
+        return self.buttons[0]
+
+    @property
+    def right_button(self) -> bool:
+        return self.buttons[2]
```

Comment:
If i was using this I'd probably start to wish these were shorter, just `left` and `right`

Reply:
By the same extension, perhaps these should be properties.